packages/expo-image/ios/EXImageView.m

```diff
@@ -79,6 +149,24 @@ - (SDExternalCompletionBlock)completionBlock
       return;
     }
 
+    // Modifications to the image like changing the resizing-mode or cap-insets
+    // cannot be handled using a SDWebImage transformer, because they don't change
+    // the image-data and this causes this "meta" data to be lost in the SDWebImage caching process.
+    if (image) {
+      if (resizeMode == RCTResizeModeRepeat) {
+        image = [image resizableImageWithCapInsets:UIEdgeInsetsZero resizingMode:UIImageResizingModeTile];
+      }
+    }
+    
+    // When no explicit source image size was specified, use the dimensions
+    // of the loaded image as the intrinsic content size.
+    if (!width && !height) {
+      [self updateIntrinsicContentSize:image.size internalAsset:NO];
```

Comment:
```suggestion
      [strongSelf updateIntrinsicContentSize:image.size internalAsset:NO];
```

Reply:
Whoops, I missed that one. Thanks, fixed it! 👍